### ISSUE TITLE
json output: option to return ref and alt allele

### ISSUE THREAD

**Issue by Hoeze** [Ensembl/ensembl-vep#731]
Hi, is there an option to return JSON with two additional items: `"ref": <str>` and `"alt": <str>` for the reference and alternative allele?
This allele string is hard to parse and match with my input file.

**Comment by dglemos** [comment]
Hello @Hoeze,
At the moment there's no option to return ref and alt alleles separately. In case there was that option, the ref and alt would be exactly the same as the allele string but splited. 
Maybe you could give us a small example of what you expect to see in the output.

**Comment by Hoeze** [comment]
I needed this as a way to join back on the input, but I did not want to parse the variant string myself.
Anyway, I found a better way by providing a unique ID in the "name" column and joining on this one.
